internal/operator/client.go

```diff
@@ -66,19 +64,28 @@ func NewClient() (Client, error) {
 		return nil, err
 	}
 
+	clientset, err := configv1.NewForConfig(kubeconfig)
+	if err != nil {
+		logger.Errorf("could not get kubernetes client")
+		return nil, err
+	}
+
 	var operatorClient Client = &operatorClient{
-		Client: client,
+		Client:        client,
+		KubeClientSet: clientset,
 	}
 	return operatorClient, nil
 }
 
 func NewPackageServerClient() (*pkgsclientv1.Clientset, error) {
 
-	cfg, err := clientcmd.BuildConfigFromFlags("", os.Getenv("KUBECONFIG"))
+	kubeconfig, err := ctrl.GetConfig()
 	if err != nil {
-		logger.Errorf("Unable to build config from flags: %w", err)
+		logger.Errorf("could not get kubeconfig")
+		return nil, err
 	}
-	pkgsclient, err := pkgsclientv1.NewForConfig(cfg)
+
+	pkgsclient, err := pkgsclientv1.NewForConfig(kubeconfig)
```

Comment:
@itroyano this is not actually a KubeClientSet and could lead to confusion. The client created here is only required within the version.go and if this refactor is required to make it more broadly used we should be more specific about the name and provide some context via a comment.